src/docs/markdown/caddyfile/directives/file_server.md

```diff
@@ -48,7 +48,7 @@ file_server [<matcher>] [browse] {
 
 - **browse** <span id="browse"/> enables file listings for requests to directories that do not have an index file.
 
-  - **<template_file>** <span id="template_file"/> is an optional custom template file to use for directory listings. Defaults to the template that can be found [here in the source code ![external link](/resources/images/external-link.svg)](https://github.com/caddyserver/caddy/blob/master/modules/caddyhttp/fileserver/browse.html). Browse templates can use actions from [the standard templates module](/docs/modules/http.handlers.templates#docs) as well.
+  - **<template_file>** <span id="template_file"/> is an optional custom template file to use for directory listings. Defaults to the template that can be extracted using the command `caddy file-server export-template`, which will print the defaut template to stdout. Browse templates can use actions from [the standard templates module](/docs/modules/http.handlers.templates#docs) as well.
```

Comment:
> I think we should also document it on the file_server directive page, for visibility.

@francislavoie, I mentioned it here. Should I also add an example at the bottom?

Reply:
Oh sorry, missed that. Mobile things.

Reply:
Actually let's keep both the original link and the command. Having two options to get it is fine.

Reply:
Done.